src/Controller/API/AcademicYears.php

```diff
@@ -36,8 +39,9 @@ public function getOne(
         }
 
         $contentTypes = $request->getAcceptableContentTypes();
+        $academicYear = $academicYearFactory->create((int) $id);
```

Comment:
I think this casting, if it is necessary, should happen in the repository. The field is an int in the DB though so I hope it isn't needed.

Reply:
i see your point.  `$id` comes from user input, as a string, but the years come from the database as integers. i moved the type casting up a bit, hopefully this will make things a bit more readable.

Reply:
:heavy_check_mark: